runtime_config_linux.go

```diff
@@ -195,7 +195,7 @@ type Network struct {
 	// Set class identifier for container's network packets
 	ClassID *uint32 `json:"classID"`
 	// Set priority of network traffic for container
-	Priorities []InterfacePriority `json:"priorities"`
+	Priorities []*InterfacePriority `json:"priorities"`
```

Comment:
@vishh I am fine one way or the other. But we have them as pointers in more places compared to where we don't.